src/modules/utility/models/User.ts

```diff
@@ -5,19 +5,19 @@ interface UserInterface {
   Username?: string;
```

Comment:
It's a matter of personal taste, but for me it is important to follow conventions within the confinement of a language, and for Javascript, Typescript properties should be camelCased, and not PascalCased :-) I know it means changing a lot in your models, but it helps on readability because PascalCasing is mostly used to declare Types e.g. User, Server etc. Just like in C# where private members most often are prefixed with '_'  and camelCased --> string _myPrivateMember , And properties are PascalCased and has auto-getters/setters. It's up to you.

Reply:
You're right, I should definitely take the time to fix this now.